Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -230,21 +236,50 @@ YUI.add('ez-binarybase-editview', function (Y) {
         },
 
         /**
+         * Checks whether the File in parameter is valid for the field. By
+         * default, only the size of the file is checked against the maximum
+         * allowed file size.
+         *
+         * @method _valid
+         * @param {File} file the File object to be stored in the field
+         * @protected
+         * @return Boolean
+         */
+        _valid: function (file) {
+            return this._validSize(file);
+        },
+
+        /**
+         * Returns the maximum allowed size in bytes or 0 if no limit is set.
+         *
+         * @method _maxSize
+         * @protected
+         * @return Number
+         */
+        _maxSize: function () {
+            var maxSize = this.get('fieldDefinition').validatorConfiguration.FileSizeValidator.maxFileSize;
+
+            return maxSize ? maxSize * 1024 * 1024 : 0;
+        },
+
+        /**
          * Checks whether the size is valid according to the field definition
-         * configuration
+         * configuration. If the file can not be accepted, a warning message is
+         * set in the `warning` attribute.
          *
-         * @param {Number} size
+         * @param {File} file the File object to be stored in the field
```

Comment:
"File" != "file" :p
(FYI "file" is parameter name, File is a build in class https://developer.mozilla.org/en-US/docs/Web/API/File)